extensions/vscode/lsp/client/src/extension.ts

```diff
@@ -0,0 +1,67 @@
+/* --------------------------------------------------------------------------------------------
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for license information.
+ * ------------------------------------------------------------------------------------------ */
+
+import { workspace, ExtensionContext } from 'vscode';
+
+import {
+	LanguageClient,
+	LanguageClientOptions,
+	ServerOptions,
+	TransportKind
+} from 'vscode-languageclient/node';
+
+
+let client: LanguageClient;
+
+export function activate(context: ExtensionContext) {
+	const serverModule = context.asAbsolutePath('/lsp-wake.wasm-cpp11-release');
+
+	// The debug options for the server
+	// --inspect=6009: runs the server in Node's Inspector mode so VS Code can attach to the server for debugging
+	let debugOptions = { execArgv: ['--nolazy', '--inspect=6009'] };

```

Comment:
I'm betting this has no effect, since our server is a wasm blob.
Perhaps we should remove the 'debug:' below as well?

Reply:
Done.